documentation/asciidoc/topics/proc_creating_grafana_datasources.adoc

```diff
@@ -0,0 +1,75 @@
+[id='creating-grafana-dashboards_{context}']
+= Creating Grafana Data Sources
+Create a `GrafanaDatasource` CR so you can visualize {brandname} metrics in Grafana dashboards.
+
+.Prerequisites
+
+* Create a Prometheus `ServiceMonitor` that scrapes {brandname} metrics.
+* Install the Grafana Operator into any namespace and then create a Grafana deployment.
```

Comment:
After the Grafana Operator installation, it's also necessary to create a Grafana CR to deploy application

Reply:
"and then create a Grafana deployment." - Does this not sound correct?

Reply:
Internally Grafana operator will create a k8s Deployment, but from the user point of view it will be Grafana custom resource